File: backend/GeoQuiz_backend/GeoQuiz_backend/Application/DTOs/User/AchievementUnlockedMessage.cs
namespace GeoQuiz_backend.Application.DTOs.User
{
    public class AchievementUnlockedMessage
    {
        public List<AchievementDto> Achievements { get; set; }
        public AchievementUnlockedMessage() { }
        public AchievementUnlockedMessage(List<AchievementDto> achievementUnlockeds) {
            Achievements = achievementUnlockeds;
        }
    }
}

File: frontend/app/src/main/java/com/example/geoquiz_frontend/data/remote/dtos/profile/AchievementUnlockedMessage.java
package com.example.geoquiz_frontend.data.remote.dtos.profile;

import java.util.List;

public class AchievementUnlockedMessage {
    private List<ProfileResponse.AchievementDto> achievements;

    public List<ProfileResponse.AchievementDto> getAchievements() {
        return achievements;
    }
}
